tests/lib/RichText/RendererTest.php

```diff
@@ -1733,43 +1734,41 @@ protected function getRepositoryMock()
         return $this->createMock(Repository::class);
     }
 
-    /**
-     * @var \Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface|\PHPUnit\Framework\MockObject\MockObject
-     */
-    protected $authorizationCheckerMock;
+    /** @var \Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface&\PHPUnit\Framework\MockObject\MockObject */
+    protected $configResolverMock;
 
     /**
-     * @return \PHPUnit\Framework\MockObject\MockObject
+     * @return \Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface&\PHPUnit\Framework\MockObject\MockObject
      */
-    protected function getAuthorizationCheckerMock()
+    protected function getConfigResolverMock()
     {
-        return $this->createMock(AuthorizationCheckerInterface::class);
+        return $this->createMock(ConfigResolverInterface::class);
     }
 
     /**
-     * @var \Psr\Log\LoggerInterface|\PHPUnit\Framework\MockObject\MockObject
+     * @var \Symfony\Component\Templating\EngineInterface|\PHPUnit\Framework\MockObject\MockObject
      */
-    protected $configResolverMock;
+    protected $templateEngineMock;
```

Comment:
```suggestion
    protected EngineInterface $templateEngineMock;
```

Reply:
@konradoboza you probably meant `Environment`.